packages/api/src/user.js

```diff
@@ -221,3 +221,49 @@ export async function userTokensDelete (request, env) {
 
   return new JSONResponse(res.deleteAuthToken)
 }
+
+/**
+ * Retrieve a page of user uploads.
+ *
+ * @param {AuthenticatedRequest} request
+ * @param {import('./env').Env} env
+ */
+export async function userUploadsGet (request, env) {
```

Comment:
nit: better `userUploadsList`? While it is paginated, a get usually ties with an id to be fetched

Reply:
Currently they are all suffixed with the HTTP method used so this is consistent with that.